kge/job/train.py

```diff
@@ -120,21 +121,17 @@ def create(
         forward_only=False,
     ) -> "TrainingJob":
         """Factory method to create a training job."""
-        if config.get("train.type") == "KvsAll":
-            return TrainingJobKvsAll(
-                config, dataset, parent_job, model=model, forward_only=forward_only
-            )
-        elif config.get("train.type") == "negative_sampling":
-            return TrainingJobNegativeSampling(
-                config, dataset, parent_job, model=model, forward_only=forward_only
-            )
-        elif config.get("train.type") == "1vsAll":
-            return TrainingJob1vsAll(
-                config, dataset, parent_job, model=model, forward_only=forward_only
-            )
-        else:
-            # perhaps TODO: try class with specified name -> extensibility
-            raise ValueError("train.type")
+        train_type = config.get("train.type")
+        class_name = config.get_default(f"{train_type}.class_name")
```

Comment:
Could this not also go into init_from in all occurrences, or am I missing something?